### DISCUSSION TITLE
Make static id on rustdesk 1.2.3

### DISCUSSION THREAD

**Discussion by Devious3901** [rustdesk/rustdesk#7595] [General]
### Bug Description

Hi, I have a problem with my user ID.
I connect via rustdesk to another device, but every time I need to enter a new id. That is, I use 1.2.1 where there is a static ID, that is, it is unchanged, and I connect to another pc, with rustdesk version 1.2.3 with a dynamic ID. that is, every time I open the program, it changes to random numbers. How can I disable this? I did not find the right item in the settings

Translated with DeepL.com (free version)

### How to Reproduce

 connect via rustdesk to another device, but every time I need to enter a new id. That is, I use 1.2.1 where there is a static ID, that is, it is unchanged, and I connect to another pc, with rustdesk version 1.2.3 with a dynamic ID. that is, every time I open the program, it changes to random numbers.

### Expected Behavior

You need to enter a new ID every time, which was not the case in other versions, and even when the rastdesk icon was blue, not white

### Operating system(s) on local side and remote side

Debian 12 / 1.2.1

### RustDesk Version(s) on local side and remote side

Devuan 5 / 1.2.3

### Screenshots

random numbers all time after restart on remote pc

**Comment by rustdesk** [comment]
It is static, but sadly not static on your machine for some unknown reason.

**Reply by rustdesk** [reply]
@fufesou follow this up.

**Comment by fufesou** [comment]
@Devious3901 Hi, is the home directory "/home/[username]" ?

If your username is "abc", is the home directory "/home/abc" ?

**Reply by Devious3901** [reply]
Hi, same directory in host and remote /home/username/Applications/
run through appimagelaucher by default

**Reply by fufesou** [reply]
Can you please run the test build here https://github.com/fufesou/rustdesk/releases/tmp

It is the latest master commit. But we can get some clues if the id changes on your machine.

It is built from https://github.com/fufesou/rustdesk/actions/runs/8533143068

The changes https://github.com/fufesou/rustdesk/commit/e4d8dad73a92552a82a7e868165f6f460d335bfe

You can run the follwing command to get the output.

```bash
journalctl --since "2 days ago" | grep "TEST ==============="
```

It will print some logs like 
```
Apr 03 01:31:19 username-1-2 rustdesk[4460]: TEST ====================== log path "/root/.local/share/logs/RustDesk/service"
```

Please hide your username when attching contents here.

**Reply by rustdesk** [reply]
AppImage?

**Comment by Devious3901** [comment]
> AppImage?
yes https://github.com/rustdesk/rustdesk/releases/download/1.2.3-1/rustdesk-1.2.3-1-x86_64.AppImage

**Comment by xlionjuan** [comment]
Latest 1.2.4 nightly may change its ID every reboot on Ubuntu for some reason, I need to downgrade to 1.2.3 to stop it regenerate new ID

And it is more funny that it will also changed when turn on or off Cloudflare WARP
